tests/end-to-end/kubeless-integration/test-kubeless.go

```diff
@@ -35,11 +35,11 @@ func deleteK8s(yamlFile string) {
 }
 
 func printContentsOfNamespace(namespace string) {
-	getResourcesCmd := exec.Command("kubectl", "-n", namespace, "get", "all,serviceinstance,servicebinding,servicebindingusage,function,subscription,api,eventactivation")
+	getResourcesCmd := exec.Command("kubectl", "-n", namespace, "get", "deploy,svc,rs,po,serviceinstance,servicebinding,servicebindingusage,function,subscriptions.eventing.kyma-project.io,api.gateway.kyma-project.io,eventactivation.applicationconnector.kyma-project.io")
```

Comment:
please use the long names, all the other resources are specified with their long names

Reply:
changed